docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -64,31 +74,114 @@ Over the duration of a given microsimulation run, simulant attributes are dynami
+
+- Surfaces complications that are more hidden in non-individual-based models (e.g., compartmental models can obscure certain limitations)
+
+Disadvantages of microsimulation
+++++++++++++++++++++++++++++++++
+
+Resource requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+Our microsimulation models require a significant amount of detailed data to accurately represent individual behaviors and the relationships between them. Obtaining this data can be challenging and time-consuming, and there may be privacy concerns associated with using such detailed personal data. 
+
+If we're not modeling the sort of complexity that microsimulation enables... perhaps a multiplication model would suit us just as well?
+
+Residual confounding
+~~~~~~~~~~~~~~~~~~~~
+
+Getting realistic, individual-level characteristics from population-level data is difficult, and creates opportunity for residual confounding...
+
+Simulant agency
+~~~~~~~~~~~~~~~
+
+Unlike agent-based models, microsimulations do not necessarily program agency into individual simulants. This removes some of the real-world likeness of these models...
 
 .. todo::
 
-  Fill out this section with strengths and weakness of individual-based microsimulation. Could also add context about how microsim compares to agent-based simulation and the broader umbrella term of IBM.
+  Fill out this section with strengths and weakness of microsimulation. Could also add context about how microsim compares to agent-based simulation and the broader umbrella term of IBM.
+
+  Explanation of relationships between macroscopic population-level measures such as incidence rate and the corresponding microscopic hazard rates we use in our simulations, and how these relationships might affect our model design and V&V.
+
+  Cite [Allen-et-al-2019]_ and [Sorensen-et-al-2017]_ somewhere.
   
 
 How does Vivarium compare with other microsimulation tools?
 -----------------------------------------------------------
 
+Leveraging the Global Burden of Disease (GBD) Study
++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Based on GBD population-level estimates stratified by age, sex, time, and location, the IHME microsimulation framework creates a simulated population that mirrors the relevant qualities of a given real-world population of interest. Each individual simulant in the population is assigned health and demographic attributes that specify characteristics such as age, sex, disease, and risk factor status (for example, diabetes diagnosis, systolic blood pressure level, or body-mass index). In any given microsimulation analysis, there can be thousands, millions, or, theoretically, billions of simulants. Although the assigned health and demographic attributes are heterogeneous across individuals within the simulated population, when taken in aggregate they match the relevant real-world population-level data supplied by the GBD.
```

Comment:
What do you mean by the "IHME microsimulation framework" (and "the IHME platform" in the next paragraph)? Are these synonyms with "Vivarium"? I'd rather just say "Vivarium" if so.